csharp/Ice/Secure/Server/Program.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) ZeroC, Inc.
+
+// Create an Ice communicator. We'll use this communicator to create an object adapter.
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+
+await using Ice.Communicator communicator = Ice.Util.initialize(ref args);
+
+// Create the authentication options using the test server certificate included with the demos.
+var sslAuthenticationOptions = new SslServerAuthenticationOptions
+{
+    ServerCertificateContext = SslStreamCertificateContext.Create(
+        X509CertificateLoader.LoadPkcs12FromFile(
+            "../../../../certs/server.p12",
+            password: "password",
+            keyStorageFlags: X509KeyStorageFlags.Exportable),
+        additionalCertificates: null)
+};
+
+// Create an object adapter that listens for incoming requests and dispatches them to servants.
+Ice.ObjectAdapter adapter = communicator.createObjectAdapterWithEndpoints(
+    "GreeterAdapter",
+    "ssl -p 4063",
```

Comment:
I would use 4061, just like in the Config demo. 4063/4 are for Glacier2. 4062 is for icerpc.